app/services/attachments/create_service.rb

```diff
@@ -36,6 +37,8 @@ def execute # rubocop:disable Metrics/CyclomaticComplexity
 
       @attachments.each(&:save)
 
+      launch_automated_workflow_executions(@pe_attachments&.last) if @attachable.instance_of?(Sample)
```

Comment:
I think instead of returning from this method `launch_automated_workflow_executions` below if the attachable project namespace doesn't have any automated workflows after checking if the pe_attachment pair has the keys (forward, reverse), we don't call this method right here if the project has no automated workflows

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/574/commits/9fe86f0187a4f2a4afff14bbe0c94111b244ef4d